### PR TITLE
Fix incorrect link to detox

### PR DESCRIPTION
**Pull request by retyui**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 React Native module to get launch arguments. Make passing parameters from testing tool to react native super easy.
 
 Mostly it's made for using
-* [`launchArgs parameter of device.launchApp method`](https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md#7-launchargsadditional-process-launch-arguments) of [Detox](https://github.com/wix/Detox/)
+* [`launchArgs parameter of device.launchApp method`]([https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md#7-launchargsadditional-process-launch-arguments](https://wix.github.io/Detox/docs/api/device/#deviceapplaunchargs)) of [Detox](https://github.com/wix/Detox/)
```
Comment (iamolegga) [inline README.md:6]: not sure how link inside link should work. maybe this:

```suggestion
* [`launchArgs parameter of device.launchApp method`](https://wix.github.io/Detox/docs/api/device/#deviceapplaunchargs) of [Detox](https://github.com/wix/Detox/)
```

Comment (retyui) [reply README.md]: fixed

**Comment by iamolegga** [comment]
thanks

**Comment by retyui** [comment]
thanks you too !

**Pull request merged**